### PR TITLE
update measures doc

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr updates some yaml code exmaples in the measure doc. raised by Zach in dbt community slack: https://getdbt.slack.com/archives/C02CCBBBR1D/p1719959209705599

Outstanding question raised to SL team:
- Can non_additive_dimensions be used with all metric types? (probably a harder question)

### CODE REVIEW

File: website/docs/docs/build/measures.md [hunk 1]
```diff
@@ -243,26 +244,26 @@ semantic_models:
             - user_id 
 
 metrics:
-  - name: mrr_end_of_month
+  - name: mrr
```
Comment (mirnawong1) [inline website/docs/docs/build/measures.md]:
```suggestion
  - name: mrr_metrics
```

File: website/docs/docs/build/join-logic.md [hunk 2]
```diff
@@ -70,16 +70,17 @@ semantic_models:
...
-
-Note that the `average_purchase_price` measure is defined in the `transactions` semantic model, where `user_id` is a foreign entity. However, the `user_signup` semantic model has `user_id` as a primary entity. 
-
-Since this is a foreign-to-primary relationship, a left join is implemented where the `transactions` semantic model joins the `user_signup` semantic model since the `average_purchase_price` measure is defined in the `transactions` semantic model.
+- MetricFlow uses`user_id` as the join key to link two semantic models, `transactions` and `user_signup`. This enables you to query the `average_purchase_price` metric in `transactions`, grouped by the `type` dimension in the `user_signup` semantic model.
+  - Note that the `average_purchase_price` measure is defined in the `transactions` semantic model, where `user_id` is a foreign entity. However, the `user_signup` semantic model has `user_id` as a primary entity. 
+- Since this is a foreign-to-primary relationship, a left join is performed where the `transactions` semantic model joins the `user_signup` semantic model since the `average_purchase_price` measure is defined in the `transactions` semantic model.
```
Comment (nataliefiann) [inline website/docs/docs/build/join-logic.md]: Hiya @mirnawong1 

The doc is great and there are no real changes or suggestions I have. The only thing I thought was maybe rewording this sentence "Since this is a foreign-to-primary relationship, a left join is performed where the `transactions` semantic model joins the `user_signup` semantic model since the `average_purchase_price` measure is defined in the `transactions` semantic model." as SL is ref'd 3 times. 

Kind Regards

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Other than the one comment I left, the PR is great and I've approved this for you. Please let me know if there's anything I can do to help with this.

Kind Regards

File: website/docs/docs/build/join-logic.md (hunk 2 above)
Comment (mirnawong1) [inline website/docs/docs/build/join-logic.md]:
```suggestion
- Since `user_id` is a foreign key in `transactions `and a primary key in `user_signup`, MetricFlow performs a left join where `transactions` joins `user_signup` to access the `average_purchase_price` measure defined in `transactions`.
```

File: website/docs/docs/build/join-logic.md [hunk 3]
```diff
@@ -70,16 +70,17 @@ semantic_models:
         type: categorical
 ```
 
-MetricFlow will use `user_id` as the join key to join two semantic models, `transactions` and `user_signup`. This enables you to query the `average_purchase_price` metric in `transactions`, sliced by the `type` dimension in the `user_signup` semantic model.
-
-Note that the `average_purchase_price` measure is defined in the `transactions` semantic model, where `user_id` is a foreign entity. However, the `user_signup` semantic model has `user_id` as a primary entity. 
-
-Since this is a foreign-to-primary relationship, a left join is implemented where the `transactions` semantic model joins the `user_signup` semantic model since the `average_purchase_price` measure is defined in the `transactions` semantic model.
+- MetricFlow uses`user_id` as the join key to link two semantic models, `transactions` and `user_signup`. This enables you to query the `average_purchase_price` metric in `transactions`, grouped by the `type` dimension in the `user_signup` semantic model.
```
Comment (mirnawong1) [inline website/docs/docs/build/join-logic.md]:
```suggestion
- MetricFlow uses `user_id` as the join key to link two semantic models, `transactions` and `user_signup`. This allows you to query the `average_purchase_price` metric in the `transactions` semantic model, grouped by the `type` dimension in the `user_signup` semantic model.
```

File: website/docs/docs/build/join-logic.md [hunk 4]
```diff
@@ -70,16 +70,17 @@ semantic_models:
...
-
-Note that the `average_purchase_price` measure is defined in the `transactions` semantic model, where `user_id` is a foreign entity. However, the `user_signup` semantic model has `user_id` as a primary entity. 
-
-Since this is a foreign-to-primary relationship, a left join is implemented where the `transactions` semantic model joins the `user_signup` semantic model since the `average_purchase_price` measure is defined in the `transactions` semantic model.
+- MetricFlow uses `user_id` as the join key to link two semantic models, `transactions` and `user_signup`. This allows you to query the `average_purchase_price` metric in the `transactions` semantic model, grouped by the `type` dimension in the `user_signup` semantic model.
+  - Note that the `average_purchase_price` measure is defined in the `transactions` semantic model, where `user_id` is a foreign entity. However, the `user_signup` semantic model has `user_id` as a primary entity.
```
Comment (mirnawong1) [inline website/docs/docs/build/join-logic.md]:
```suggestion
  - Note that the `average_purchase_price` measure is defined in `transactions`, where `user_id` is a foreign entity. However, `user_signup` has `user_id` as a primary entity. 
```

File: website/docs/docs/build/join-logic.md [hunk 5]
```diff
@@ -70,16 +70,17 @@ semantic_models:
...
-
-Note that the `average_purchase_price` measure is defined in the `transactions` semantic model, where `user_id` is a foreign entity. However, the `user_signup` semantic model has `user_id` as a primary entity. 
-
-Since this is a foreign-to-primary relationship, a left join is implemented where the `transactions` semantic model joins the `user_signup` semantic model since the `average_purchase_price` measure is defined in the `transactions` semantic model.
+- MetricFlow uses `user_id` as the join key to link two semantic models, `transactions` and `user_signup`. This allows you to query the `average_purchase_price` metric in the `transactions` semantic model, grouped by the `type` dimension in the `user_signup` semantic model.
+  - Note that the `average_purchase_price` measure is defined in `transactions`, where `user_id` is a foreign entity. However, `user_signup` has `user_id` as a primary entity. 
+- Since `user_id` is a foreign key in `transactions `and a primary key in `user_signup`, MetricFlow performs a left join where `transactions` joins `user_signup` to access the `average_purchase_price` measure defined in `transactions`.
```
Comment (mirnawong1) [inline website/docs/docs/build/join-logic.md]:
```suggestion
- Since `user_id` is a foreign key in `transactions` and a primary key in `user_signup`, MetricFlow performs a left join where `transactions` joins `user_signup` to access the `average_purchase_price` measure defined in `transactions`.
```

File: website/docs/docs/build/join-logic.md [hunk 6]
```diff
@@ -70,16 +70,17 @@ semantic_models:
...
-
-Note that the `average_purchase_price` measure is defined in the `transactions` semantic model, where `user_id` is a foreign entity. However, the `user_signup` semantic model has `user_id` as a primary entity. 
-
-Since this is a foreign-to-primary relationship, a left join is implemented where the `transactions` semantic model joins the `user_signup` semantic model since the `average_purchase_price` measure is defined in the `transactions` semantic model.
+- MetricFlow uses `user_id` as the join key to link two semantic models, `transactions` and `user_signup`. This allows you to query the `average_purchase_price` metric in the `transactions` semantic model, grouped by the `type` dimension in the `user_signup` semantic model.
+  - Note that the `average_purchase_price` measure is defined in `transactions`, where `user_id` is a foreign entity. However, `user_signup` has `user_id` as a primary entity. 
+- Since `user_id` is a foreign key in `transactions` and a primary key in `user_signup`, MetricFlow performs a left join where `transactions` joins `user_signup` to access the `average_purchase_price` measure defined in `transactions`.
```
Comment (mirnawong1) [inline website/docs/docs/build/join-logic.md:75]: thanks @nataliefiann ! updated/reworded this!

**Review by Jstein77 — APPROVED** [review]
LGTM!
